### ISSUE TITLE
When browsing HTML in the LBRY desktop app, a back button would be handy

### ISSUE THREAD

**Issue by eggplantbren** [lbryio/lbry-desktop#2547]
This feature request is fairly minor in importance, but also shouldn't be very hard to implement.

I've uploaded an archive of an old web site to LBRY (lbry://bayes-wustl-edu) and will soon be uploading another. You can browse it in the app, which is awesome, but there's no back button there, which also makes it fairly useless depending on how the site is designed.

The comic book viewer has a little toolbar with navigation stuff on it, and I thought something similar would suffice for viewing HTML files.

**Comment by seanyesmunt** [comment]
Good idea. I think we can do something like this https://stackoverflow.com/questions/3254985/back-and-forward-buttons-in-an-iframe
